frontend/src/app/_components/creator-card/creator-card.component.html

```diff
@@ -0,0 +1,10 @@
+<div  class="creator card">
+    <img [routerLink]="'/' + component + '/' + creator.id" src="{{creator.images[0].url}}" class="card-img-top" alt="{{creator.images[0].alt}}" />
```

Comment:
Would routerLink="/{{component}}/{{creator.id}}" be more readable (and would it work?)

Reply:
Actually, at the same time, wouldn't [src]="creator.images[0].url" be more readable (and similarly for alt?)

Reply:
routerLink doesn't seem to take {{}} well, i had tried that one and it was not happy.

i'll change everything re the second comment